pages/partials/questionFooter.ejs

```diff
@@ -25,6 +25,8 @@
                <% } else if (variantAttemptsLeft == 1) { %>
                <small class="font-italic ml-2">Single attempt</small>
                <% } %>
+            <% } else if (question_context != 'instructor') { %>
+               <small class="font-italic ml-2">Unlimited attempts</small>
```

Comment:
I think we also need this change in the non-Freeform question section down below (yuck! I'm sorry! 😢).

Reply:
Done, though I don't have these kinds of questions to test them. The freeform ones are not broken, though, so that's good :-D